hnn_core/viz.py

```diff
@@ -1255,19 +1264,40 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
         The matplotlib figure handle.
     """
     import matplotlib.pyplot as plt
+    from scipy.interpolate import RectBivariateSpline
+
     if ax is None:
         _, ax = plt.subplots(1, 1, constrained_layout=True)
 
-    im = ax.pcolormesh(times, contact_labels, np.array(data),
-                       cmap="jet_r", shading='auto')
-    ax.set_title("CSD")
+    if sink[0] == 'b':
+        cmap = "jet"
+    elif sink[0] == 'r':
+        cmap = "jet_r"
```

Comment:
what if the user passes 'B' ? I have seen people doing things like that ...

Reply:
you can do something like:

```py
if sink[0].tolower() == 'b'
```

but in addition, also throw an error if the string is something other than 'r' or 'b'